spaceship/lib/spaceship/du/utilities.rb

```diff
@@ -7,6 +7,7 @@ module Utilities #:nodoc:
     # Identifies the content_type of a file based on its file name extension.
     # Supports all formats required by DU-UTC right now (video, images and json)
```

Comment:
This is not correct any more now. Is there a documentation from Apple that lists all the allowed file types you added here?

Reply:
I have taken this list from the submission page.

![Screenshot 2019-03-19 at 12 48 17 AM](https://user-images.githubusercontent.com/8597258/54528130-e8474e80-49e0-11e9-97f7-9990a42ce319.png)

Reply:
Oh great. 

Can we update the comment here and link to that page/list (or describe how to get to it and find the list) or find another source for it?

Reply:
yes updated....